sdk/core/core-tracing/src/useInstrumenter.browser.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NoOpInstrumenter } from "./instrumenter";
+import { Instrumenter } from "./interfaces";
+
+const GLOBAL_INSTRUMENTER_SYMBOL = Symbol("@azure/core-tracing instrumenter");
+
+interface Cache {
+  instrumenter: Instrumenter;
+}
+
+let cache: Cache;
+function getCache(): Cache {
+  if (!cache) {
+    const globalObj: any = globalThis;
+    if (!globalObj[GLOBAL_INSTRUMENTER_SYMBOL]) {
+      globalObj[GLOBAL_INSTRUMENTER_SYMBOL] = {
+        instrumenter: new NoOpInstrumenter()
+      };
+    }
+    cache = globalObj[GLOBAL_INSTRUMENTER_SYMBOL];
+  }
+  return cache;
+}
```

Comment:
do we need any of this? It seems like the browser version just hooks `.enable()` now instead of placing the Instrumenter on the global object?

Reply:
My original thought here was to support side-by-side installs of core-tracing; however, as discussed offline going to punt it as YAGNI and make an issue to just ensure that scenario works as expected 👍

Reply:
Ok, I removed the browser-specific global crud and created https://github.com/Azure/azure-sdk-for-js/issues/18866 to track that we ensure side-by-side works as expected (or at least have a conversation about it)